dev/ci.sh

```diff
@@ -257,6 +258,11 @@ GAPInput
     # TODO: should we install the GAP test suite???
     cp -R $SRCDIR/tst $GAPPREFIX/share/gap/
     $GAPPREFIX/bin/gap $GAPPREFIX/share/gap/tst/testinstall.g
+
+    # test integration with pkg-config
+    cd "$SRCDIR"
+    make tst/testpkgconfigversion
+    make tst/testpkgconfigbuild
```

Comment:
```suggestion
    make testpkgconfigversion
    make testpkgconfigbuild
```

Reply:
done in [2bd19b0](https://github.com/gap-system/gap/pull/5080/commits/2bd19b0d2075de2ef0fbcb9bd6232ebfa2b48236) and [c0b8513](https://github.com/gap-system/gap/pull/5080/commits/c0b851399729734486fb6694930fd5b810deba9a)